strawberryfields/plot.py

```diff
@@ -120,3 +123,235 @@ def generate_wigner_chart(data, xvec, pvec, contours=True):
     chart["layout"]["scene"]["zaxis"]["title"] = "W(x,p)"
 
     return chart
+
+
+# Plot.ly default barchart JSON
+barchart_default = {
```

Comment:
Is all of this directly copied from SF interactive? I decided to _not_ use the default chart for the Wigner plotting, and simply used a slightly more minimal dict defined directly in `generate_wigner_chart` instead. I think that quite a few of these options either didn't make sense or didn't seem to change anything (so I removed them in favour of a more streamlined set of plotting options). :thinking:

Reply:
Yes, it's the one used online atm (except for a couple of options which became invalid). Would there be any options that are worth leaving out?

Reply:
Not really. They actually all seem nice to have here. Just played around with them a bit, and I think they make sense to keep.